wagtail/admin/views/generic/mixins.py

```diff
@@ -710,13 +710,7 @@ def get_editing_sessions(self):
                 "wagtailadmin_editing_sessions:release",
                 args=(session.id,),
             ),
-            list(
```

Comment:
See commit messages for details, but the reason why we don't load the editing sessions immediately on initial render is:
- To prevent race condition with the release endpoint being called when you reload the page i.e. the session is not removed yet when the reload request is sent, causing the previous session to appear initially
- To avoid having to duplicate/extract the logic in the  ping view (annotating the latest revision and sorting the list etc.)

It also helps with performance as it means there's one less query to be performed on initial page render from the server.